server/mvc/models/User.js

```diff
@@ -8,29 +8,21 @@ var config,
 
 module.exports = function(){
   var self = this,
-    document = new ThinkyModel({}),
-    exists = false;
+    document = new ThinkyModel({});
 
   this.findByUserId = function(userId){
```

Comment:
I think getByUserId (or getById) makes more sense here, since it is doing a retrieval rather than an actual search.